src/sidebar/components/ShareDialog/test/ExportAnnotations-test.js

```diff
@@ -66,9 +68,12 @@ describe('ExportAnnotations', () => {
       '../../store': { useSidebarStore: () => fakeStore },
     });
 
-    // Restore this very simple component to get it test coverage
     $imports.$restore({
+      // Restore this very simple component to get it test coverage
       './LoadingSpinner': true,
+      // Restore UserAnnotationsListItem, as it's used as some buttons' content
+      // and can make a11y tests fail
```

Comment:
```suggestion
      // and is needed to make a11y tests pass
```

The current wording suggests that `UserAnnotationsListItem` causes a11y tests to fail, but actually it is needed for them to pass.